packages/plugin-reconstruction/src/index.spec.ts

```diff
@@ -0,0 +1,175 @@
+import { jest } from "@jest/globals";
+import { clickTarget, pressKey, startTimeline } from "jspsych/tests/utils";
+
+import reconstruction from ".";
+
+jest.useFakeTimers();
+
+describe("reconstruction", () => {
+  test("default starting value renders correctly", async () => {
+    let timeline = [
```

Comment:
`const`? Style question: Assign the timeline to a variable or not? My rule of thumb is always: Do I need it more than once? If not, does it substantially improve code readability? In this case, for me, it's pretty obvious that the parameter to `startTimeline()` is a timeline (and all tests look the same), so I did not assign it to a variable first in my tests...

Reply:
Good point, maybe I'm just too used to that. What about the tests that I wrote – would you like me to take the `const timeline` approach as well there? Also, regarding readability, it may be helpful to add a `startTrial` method that wraps `startTimeline` and takes a single trial instead of the timeline (to get rid of the wrapping `[]` everywhere). What do you think about that?

Reply:
I guess my preference is that the tests look like experiments as much as possible, so I'd rather explicitly have a timeline and avoid too many test-specific utilities. 

But I don't mind if our test files look different, at least for now. My bigger priority is just moving towards a stable-enough  suite to release 7.0.

Reply:
I agree about the extra function, less is more. Ok!